sros/docker/launch.py

```diff
@@ -758,10 +762,18 @@ def __init__(self, hostname, username, password, mode, variant_name, conn_mode):
                     conn_mode,
                 )
             ]
+
+            start_eth = 1
             for i, lc in enumerate(variant["lcs"]):
+                # Priority is to use max_nics from each lc definition
+                max_nics = lc.get('max_nics', None)
+                if not max_nics:
+                    max_nics = variant['max_nics']
+
                 self.vms.append(
-                    SROS_lc(lc, conn_mode, variant["max_nics"], slot=2 + i),
+                    SROS_lc(lc, conn_mode, max_nics, slot=2 + i, nic_eth_start=start_eth)
```

Comment:
interesting, why do we use `2` as a base for a slot number? I think it should've been `1+i`

Reply:
I think it is to ensure that overlay qcow2 image naming and qemu instance monitoring port still in following sequence: CPM slot A, CPM slot B, LC Slot 1, LC Slot 2 and so on.